statsmodels/tsa/statespace/tests/test_mlemodel.py

```diff
@@ -597,6 +597,23 @@ def test_summary():
         res.summary()
 
 
+def test_summary_rsquared():
+    from statsmodels.regression.linear_model import OLS
+    endog = np.array([1, 2, 4, 8, 16])
+    exog = np.array([1, 2, 4, 8, 16])
```

Comment:
This shouldn't be a perfect fitting model, so you should make exog not identical to endog.  Also, so that the OLS R^2 matches, exog needs to include a constant column.